master/pom.xml

```diff
@@ -75,7 +75,6 @@
   <properties>
     <sakai.version>11-SNAPSHOT</sakai.version>
     <!-- kernel only versions -->
-    <sakai.kernel.version>11-SNAPSHOT</sakai.kernel.version>
```

Comment:
To make contrib tools work without having to change could you just do:
```
<sakai.kernel.version>${sakai.version}</sakai.kernel.version>
```
so they are always in sync.